src/versions.js

```diff
@@ -48,6 +48,11 @@ const versions = {
                     designLibraryVersion: '2.0.0',
                     designLibraryDoc: 'https://www.sketch.com/s/6c8808fa-bf58-42e5-a0be-7e9d5798fc5f/p/45DC2CF5-702E-4572-9EDF-D952D3A22727',
                 },
+                timePicker: {
+                    devLibraryVersion: '???',
```

Comment:
Please use:

```
timePicker: {
    devLibraryVersion: '2.0.0',
    designLibraryVersion: 'N/A',
    designLibraryDoc: 'N/A',
}
```

Reply:
Made it so! https://github.com/saddlebackdev/react-cm-ui/pull/319/commits/f8a15e4a42c12c4798a3deca1e2b39ad21058ff4